internal/pkg/config/config.go

```diff
@@ -0,0 +1,89 @@
+package config
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/joshsoftware/peerly-backend/internal/pkg/apperrors"
+	"github.com/spf13/viper"
+)
+
+var (
+	appName                string
+	appPort                int
+	jwtKey                 string
+	jwtExpiryDurationHours int
+)
+
+// Load - loads all the environment variables and/or params in application.yml
+func Load(configFile string) {
+	viper.SetDefault("APP_NAME", "app")
```

Comment:
use consts instead of hard-coded strings for env var name. applicable everywhere in your code.